pages/user guides/discover_nmap.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Discover your Hosts with Nmap
+keywords: nmap, ip, address, api, services
+last_updated: September 2020
+summary: "How to use nmap to scan your network"
+sidebar: main_sidebar
+permalink: discover-nmap.html
+---
+
+
+
+As part of your cloud migration journey it is important to have all the tools at your disposal. In addition to our proprietary tools, you can now use [Nmap](https://nmap.org/) to facilitate your discovery process. 
+The way it works is very simple and with the full power of nmap available to you, discovery of even those hard to reach places is possible. 
+ 
+Nmap has developed some powerful capabilities and good [documentation](https://nmap.org/book/host-discovery-find-ips.html) that will allow you to identify  all the hosts you have running on your network, the ports and services that they have exposed and even some vulnerabilities discovered by the open source community. 
+ 
+Nmap is capable of producing its output in an XML file, allowing you to inspect the raw scan output before sending it to the Tidal Migrations API with our CLI,  [Tidal Tools](https://get.tidal.sh). Once uploaded to the Tidal Migrations Platform, you will be able to visualize your network devices, track your complete server inventory and build on this data with other discovery methods, allowing you to make informed decisions on your cloud migration path.
+
+
+## Using Nmap with Tidal Tools
+ 
+By leveraging the power of **Tidal Tools** you can send the output generated by Nmap to Tidal migration API.
+ 
+To get the process started your must follow these steps first:
```

Comment:
```suggestion
```

(seems redundant, and it's more than getting started, it's how to do it end to end.)